R/ggplotly.R

```diff
@@ -552,7 +552,7 @@ gg2list <- function(p, width = NULL, height = NULL,
       )
       # allocate enough space for the _longest_ text label
       axisTextX <- theme[["axis.text.x"]] %||% theme[["axis.text"]]
-      labz <- unlist(lapply(layout$panel_params, "[[", "x.labels"))
+      labz <- unlist(lapply(layout$panel_params, function(pp) pp[["x"]]$get_labels %()% pp$x.labels))
```

Comment:
use `{``}`

Reply:
Why, just for style?

Reply:
is it `(function(x) x) %()% y` or `function(x) { x %()% y}` ?

Hard to read. So I guess style.